app/models/concerns/host_common.rb

```diff
@@ -32,7 +32,6 @@ module HostCommon
     has_many :lookup_values, :primary_key => :lookup_value_matcher, :foreign_key => :match, :dependent => :destroy
     # See "def lookup_values_attributes=" under, for the implementation of accepts_nested_attributes_for :lookup_values
     accepts_nested_attributes_for :lookup_values
-    has_many :lookup_keys, :through => :lookup_values
```

Comment:
I added it during an attempt to change the lookup key from the lookup value in #6007 and noticed it when working on this.
It's unused but I don't mind keeping it if you prefer.